src/js/components/ServicesTable.js

```diff
@@ -18,6 +16,10 @@ import {Table} from 'reactjs-components';
 import TableUtil from '../utils/TableUtil';
 var Units = require('../utils/Units');
 
+const StatusMapping = {
+  'Running': 'color-1'
```

Comment:
What do you think about changing the class name to something more descriptive? I know it should be abstract but perhaps `progressColor-1`